app/services/bulk_upload/sales/year2022/row_parser.rb

```diff
@@ -524,18 +528,18 @@ def attributes_for_log
 
     attributes["ethnic_group"] = ethnic_group_from_ethnic
     attributes["ethnic"] = field_30
-    attributes["national"] = field_31
-    attributes["income1nk"] = field_32.present? ? 0 : 1
+    attributes["national"] = buyer_not_interviewed? ? 13 : field_31
+    attributes["income1nk"] = buyer_not_interviewed? || field_32.blank? ? 1 : 0
     attributes["income1"] = field_32
     attributes["income2nk"] = field_33.present? ? 0 : 1
     attributes["income2"] = field_33
-    attributes["inc1mort"] = field_34
+    attributes["inc1mort"] = field_32.blank? ? 2 : field_34
     attributes["inc2mort"] = field_35
-    attributes["savingsnk"] = field_36.present? ? 0 : 1
+    attributes["savingsnk"] = buyer_not_interviewed? || field_36.blank? ? 1 : 0
     attributes["savings"] = field_36
```

Comment:
Do we perhaps want to set `savings` to be nil if the buyer hasn't been interviewed?

Reply:
same as above